src/Psalm/Config.php

```diff
@@ -740,6 +745,11 @@ private static function fromXmlAndPaths(string $base_dir, string $file_contents,
             $config->error_baseline = $attribute_text;
         }
 
+        if (isset($config_xml['includePhpVersionsInErrorBaseline'])) {
+            $attribute_text = (string) $config_xml['includePhpVersionsInErrorBaseline'];
+            $config->include_php_versions_in_error_baseline = $attribute_text === 'true' || $attribute_text === '1';
+        }
+
```

Comment:
I think you can replace the five lines above with one in the `$booleanAttributes` array [above](https://github.com/vimeo/psalm/pull/1936/files#diff-20a62f59764c9aa89c920a98d1b6ec16R658), with no loss of clarity.

Reply:
wasn't too sure about that- have implemented as suggested in 0c523ae